samples/Sandbox/MainWindow.axaml

```diff
@@ -1,4 +1,5 @@
 <Window xmlns="https://github.com/avaloniaui"
         xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'
         x:Class="Sandbox.MainWindow">
+  <TextBlock Text="random change"/>
```

Comment:
Looks like you accidentally committed a change to sandbox.

Reply:
That was just some hack to produce a new build